### DISCUSSION TITLE
Puppeteer old Headless deprecation warning:

### DISCUSSION THREAD

**Discussion by fred-fowler** [harlan-zw/unlighthouse#82] [Q&A] [answered]
Hi,  any Idea on this?!
  It's throwing an error in following
  Puppeteer old Headless deprecation warning:
    In the near feature `headless: true` will default to the new Headless mode
    for Chrome instead of the old Headless implementation. For more
    information, please see https://developer.chrome.com/articles/new-headless/.
    Consider opting in early by passing `headless: "new"` to `puppeteer.launch()`
    If you encounter any bugs, please report them to https://github.com/puppeteer/puppeteer/issues/new/choose.

**Comment by harlan-zw** [comment]
Can confirm that I've started getting this as well, will investigate.

**Comment by harlan-zw** [comment, marked as answer]
It seems like this is fixed in 0.7.0. However, we this may pop back up as we can't use the `new` headless as it causes issues with Lighthouse screenshots.

Will close for now but let me know if it pops back up.
